featuretools/primitives/standard/transform/natural_language/upper_case_word_count.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np
+from woodwork.column_schema import ColumnSchema
+from woodwork.logical_types import Double, NaturalLanguage
+
+from featuretools.primitives.base import TransformPrimitive
+
+
+class UpperCaseWordCount(TransformPrimitive):
+    """Determines the number of words in a string that are entirely capitalized.
+
+    Description:
+        Given list of strings, determine the number of words in each string
+        that are entirely capitalized.
+
+        If a string is missing, return `NaN`.
+
+    Examples:
+        >>> x = ['This IS a string.', 'This is a string', 'AAA']
+        >>> upper_case_word_count = UpperCaseWordCount()
+        >>> upper_case_word_count(x).tolist()
+        [1.0, 0.0, 1.0]
+    """
+
+    name = "upper_case_word_count"
+    input_types = [ColumnSchema(logical_type=NaturalLanguage)]
+    return_type = ColumnSchema(logical_type=Double, semantic_tags={"numeric"})
+    default_value = 0
+
+    def get_function(self):
+        pattern = r"(\w[A-Z0-9]+\b)"
```

Comment:
```suggestion
        pattern = r"(\w[A-Z0-9]+\w)"
```
I think it has to be `\w` on both sides

Reply:
Good catch, fixed:
https://github.com/alteryx/featuretools/pull/2460/commits/bc7ed24198349a335d22f5605928dac8c680970c
https://github.com/alteryx/featuretools/pull/2460/commits/206a278886dba5b9a7c86e1c20ac793c60718917